vuepress/docs/docs/README.md

```diff
@@ -21,6 +21,7 @@ To use `caliban`, add the following dependency to your `build.sbt` file:
 The following modules are optional:
 
 ```scala
+"com.github.ghostdogpr" %% "caliban-quick"      % "2.4.2" // quick adapter
```

Comment:
Maybe add in the comment: "if you're just getting started, only import that one"